tc/app/helpers/commits_summary.usage

```diff
@@ -0,0 +1,2 @@
+
+{cli command}
```

Comment:
it is necessary to describe what the command is doing 
for example - `Gets commits a summary from branch`